doc/input-output-reference/src/overview/group-location-climate-weather-file-access.tex

```diff
@@ -1831,6 +1847,13 @@ \subsubsection{Inputs}\label{inputs-23-002}
 
 Name of a schedule defined elsewhere that describes the rate of irrigation. The values in this schedule are the average rate of irrigation in meters per hour.
 
+\paragraph{Field: Use Precipitation}\label{field-use-precipitation}
```

Comment:
This needs to be deleted.

Reply:
Thanks for spotting this, I just deleted the "Use Precipitation" related documentation.